.github/workflows/ci-test-subgraph.yml

```diff
@@ -0,0 +1,89 @@
+name: CI Test Subgraph Locally
+
+on:
+  workflow_dispatch:
```

Comment:
can we make this part of the ci.feature.yml instead: https://github.com/superfluid-finance/protocol-monorepo/blob/9af63906cbdfcc43994df25e9d875319a215b423/.github/workflows/ci.feature.yml#L128

this ci-test-subgraph.yml is not something we need

Reply:
added and working now